hack/make-rules/tools.mk

```diff
@@ -1,5 +1,29 @@
+INSTALL_TOOLS += $(TOOLBIN)/yq
 $(TOOLBIN)/yq:
 	cd $(TOOLS_DIR); ./install_yq.sh
+	$(call post-install-check)
 
+INSTALL_TOOLS += $(TOOLBIN)/helm
 $(TOOLBIN)/helm:
 	cd $(TOOLS_DIR); ./install_helm.sh
+	$(call post-install-check)
```

Comment:
The call to post-install-check is not needed here (post-install-check is not present in this function)